exercises/concept/temperature/.meta/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "blurb": "Use number type conversion",
```

Comment:
The blurb should be a short description of the exercise, but one with a slightly more verbose description. So maybe something like:

```suggestion
  "blurb": "Learn about numbers and type conversion by converting temperatures",
```